src/adic_space.lean

```diff
@@ -263,29 +267,120 @@ begin
 end
 
 
-lemma stalk_map_comp' {X : Type*} [topological_space X]
-  {Y : Type*} [topological_space Y] {Z : Type*} [topological_space Z]
+lemma stalk_map_comp' {X : Type u} [topological_space X]
+  {Y : Type u} [topological_space Y] {Z : Type u} [topological_space Z]
   {F : presheaf_of_rings X}
   {G : presheaf_of_rings Y} {H : presheaf_of_rings Z}
   (a : presheaf_of_rings.f_map F G) (b : presheaf_of_rings.f_map G H) (x : X) :
   stalk_map (presheaf_of_rings.f_map_comp a b) x =
   (stalk_map a x) ∘ (stalk_map b (a.f x)) := by ext; apply stalk_map_comp
 
-structure 𝒞.map {X : Type*} [topological_space X] {Y : Type*} [topological_space Y]
+structure 𝒞.map {X : Type u} [topological_space X] {Y : Type u} [topological_space Y]
   (F : 𝒞 X) (G : 𝒞 Y) :=
```

Comment:
There's a C. We now don't need this, right?

Reply:
Removed